src/main/features/response/views/statement-of-means/check-and-send.njk

```diff
@@ -27,9 +27,16 @@
       {% set years = statementOfMeans.unemployed.unemploymentDetails.years %}
       {% set months = statementOfMeans.unemployed.unemploymentDetails.months %}
 
-      {% set status = statementOfMeans.unemployed.option.displayValue + ' for ' %}
-      {% set status = status + singularOrPlural(years, 'year') if years %}
-      {% set status = status + singularOrPlural(months, 'month') if months %}
+      {% set status = statementOfMeans.unemployed.option.displayValue %}
+      {% if years or months %}
+        {% set status = status + ' for ' %}
+        {% if years %}
+          {% set status = status + singularOrPlural(years, 'year') %}
```

Comment:
I think a filter would be more appropriate than a macro for adding the s
REC

Reply:
Good idea.

Do you know if we have any filter like this?

Reply:
not like this but we have the date filter, they are really easy to write, just one tiny typescript file